stdlib/builtins.pyi

```diff
@@ -937,6 +938,8 @@ class bool(int):
     @overload
     def __rxor__(self, __value: int) -> int: ...
     def __getnewargs__(self) -> tuple[int]: ...
+    @deprecated("Will throw an error in Python 3.14. Use `not` for logical negation of bools instead.")
```

Comment:
This is `~True`. It always behaved confusingly, so it seems fine to warn about it on all versions.